### PR TITLE
*: merge experimental-feature.md to basic-features.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Merge `experimental-features.md` to `basic-features.md` because they have much in common and need extra efforts for update. Some features in `experimental-features.md` are missing in `basic-features.md`, and these features are added to `basic-features.md`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12687 https://github.com/pingcap/docs-cn/pull/12762
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -38,8 +42,11 @@ This document lists the features supported in each TiDB version. Note that suppo
...
 | [Columnar storage (TiFlash)](/tiflash/tiflash-overview.md)   | Y | Y | Y | Y | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
+| [Use FastScan to accelerate queries in OLAP scenarios](/develop/dev-guide-use-fastscan.md) | Experimental | Experimental | Experimental | Experimental | N | N | N | N | N | N | N | N |
 | [RocksDB engine](/storage-engine/rocksdb-overview.md)        | Y | Y | Y | Y | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
 | [Titan plugin](/storage-engine/titan-overview.md)            | Y | Y | Y | Y | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge实验功能)   |  Experimental   |  Experimental   |  Experimental   |  Experimental    |  Experimental    |  Experimental    |    Experimental     |    Experimental     |    Experimental     |    Experimental     |    Experimental     |    Experimental     |
```
Comment (TomShawn) [inline basic-features.md:48]:
```suggestion
| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge-experimental)   |  Experimental   |  Experimental   |  Experimental   |  Experimental    |  Experimental    |  Experimental    |    Experimental     |    Experimental     |    Experimental     |    Experimental     |    Experimental     |    Experimental     |
```

File: basic-features.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
 ---
 title: TiDB Features
-summary: Learn about the basic features of TiDB.
-aliases: ['/docs/dev/basic-features/']
+summary: Learn about the feature overview of TiDB.
+aliases: ['/docs/dev/basic-features/','/tidb/dev/experimental-features-4.0/']
```
Comment (qiancai) [inline basic-features.md:4]: please update the aliases for stable.

File: basic-features.md [hunk 3]
```diff
@@ -67,13 +74,16 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [TiFlash Query Result Materialization](/tiflash/tiflash-results-materialization.md) | Experimental | N | N | N | N | N | N | N | N | N | N | N |
 
 ## Advanced SQL features
 
 | Advanced SQL features                                    | 6.5 | 6.4 | 6.3 | 6.2 | 6.1 | 6.0 | 5.4          |   5.3    |   5.2    |   5.1    |   5.0   |   4.0    |
 | ------------------------------------------------------------ | :--: | :--: | :--: | :--: | :--: | :--: | ------------ | :----------: | :----------: | :----------: | :----------: | :----------: |
 | [Prepared statement cache](/sql-prepared-plan-cache.md)       | Y | Y | Y | Y | Y | Y | Y            |      Y       | Experimental | Experimental | Experimental | Experimental |
 | [SQL plan management (SPM)](/sql-plan-management.md)         | Y | Y | Y | Y | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
+| [Create binding according to historical execution plan](/sql-plan-management.md#create-a-binding-according-to-a-historical-execution-plan) | Experimental | N | N | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:88]:
```suggestion
| [Create bindings according to historical execution plans](/sql-plan-management.md#create-a-binding-according-to-a-historical-execution-plan) | Experimental | N | N | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
 ---
 
 # TiDB Features
 
-This document lists the features supported in each TiDB version. Note that supports for experimental features might change before the final release.
+This document lists the features supported in each TiDB version.
+
+> **Note:**
+>
+> - Y: the feature is in general availability (GA) and can be used in production environments.
```
Comment (qiancai) [inline basic-features.md:13]:
```suggestion
> - Y: the feature is generally available (GA) and can be used in production environments.
```

File: basic-features.md [hunk 5]
```diff
@@ -1,12 +1,18 @@
...
+> - N: the feature is not supported.
+> - Experimental: the feature is not GA yet and you need to be aware of the usage limitations. Experimental features are subject to change or removal without prior notice. The syntax and implementation might change before the general availability. If you find any issue, file a [GitHub issue](https://github.com/pingcap/tidb/issues).
```
Comment (qiancai) [inline basic-features.md:15]:
```suggestion
> - Experimental: the feature is not GA yet and you need to be aware of the usage limitations. Experimental features are subject to change or removal without prior notice. The syntax and implementation might be modified before the general availability. If you find any issue, file a [GitHub issue](https://github.com/pingcap/tidb/issues).
```

File: basic-features.md [hunk 6]
```diff
@@ -1,12 +1,18 @@
...
 # TiDB Features
 
-This document lists the features supported in each TiDB version. Note that supports for experimental features might change before the final release.
+This document lists the features supported in each TiDB version.
+
+> **Note:**
+>
+> - Y: the feature is generally available (GA) and can be used in production environments.
+> - N: the feature is not supported.
+> - Experimental: the feature is not GA yet and you need to be aware of the usage limitations. Experimental features are subject to change or removal without prior notice. The syntax and implementation might be modified before the general availability. If you find any issue, file a [GitHub issue](https://github.com/pingcap/tidb/issues).
```
Comment (qiancai) [inline basic-features.md:15]:
```suggestion
> - Experimental: the feature is not GA yet and you need to be aware of the usage limitations. Experimental features are subject to change or removal without prior notice. The syntax and implementation might be modified before the general availability. If you encounter any problems, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Pull request merged**